job-server-api/src/main/scala/spark/jobserver/context/SparkContextFactory.scala

```diff
@@ -130,12 +130,12 @@ trait JavaContextFactory extends SparkContextFactory {
 }
 
 /**
- * The default factory creates a standard SparkContext.
- * In the future if we want to add additional methods, etc. then we can have additional factories.
- * For example a specialized SparkContext to manage RDDs in a user-defined way.
- *
- * If you create your own SparkContextFactory, please make sure it has zero constructor args.
- */
+  * The default factory creates a standard SparkContext.
```

Comment:
Maybe not a big deal, but I would separate out the style-related changes to make it easier to review....  this could be because I'm just viewing all the changes at once, but that's because I see some commits which are revising earlier ones.